Source/WarpX.H

```diff
@@ -236,6 +236,12 @@ public:
     void EvolveE (int lev, PatchType patch_type, amrex::Real dt);
     void EvolveF (int lev, PatchType patch_type, amrex::Real dt, DtType dt_type);
 
+    // Maxwell Hybrid QED Solver components
+    void Hybrid_QED_Push (         amrex::Vector<amrex::Real> dt);
+    void Hybrid_QED_Push (int lev, amrex::Real dt);
```

Comment:
```suggestion
    /** \brief apply QED correction on electric field for level lev
     * \param dt time step
     */
    void Hybrid_QED_Push (int lev, amrex::Real dt);
```

Reply:
This is missing the lev input? Also, do you want to define it here and not where the actual function is defined?